src/coffee/renuo_cms_client.coffee

```diff
@@ -0,0 +1,105 @@
+'use strict'
+
+class RenuoCmsClient
+
+  constructor: ->
+    @initializeContentBlocks =>
+      $('a.edit').click =>
+        # TODO: Fix this hack!
+        `var editLink = this`
+        @getContentBlockForEdit $(editLink).parent().parent().attr('data-block'), =>
+          $('a.save').click =>
+            `var saveLink = this`
+            newContent = $(saveLink).parent().parent().find('textarea').val()
+            blockID = $(saveLink).parent().parent().attr('data-block')
+            @updateContentBlock blockID, newContent
+
+  getApiKey: ->
+    'Token token=772a91a136caa729fb8e09277c05310e'
+
+  getApiUrl: ->
+    'http://localhost:3000/api/'
+
+  initializeContentBlocks: (callback) ->
+    $.ajax
+      type: 'GET'
+      url: @getApiUrl() + 'content_blocks.json'
+      beforeSend: (xhr) =>
+        xhr.setRequestHeader 'Authorization', @getApiKey()
+      dataType: 'json'
+      success: (data) =>
+        $.each data, (index, value) =>
+          console.log value
```

Comment:
Don't commit console.log in code. If you want to introduce logging, introduce a logger function and parametrize from above.